minjoo/app.js

```diff
@@ -0,0 +1,67 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch(() => {
+    console.log("Promise Rejected!");
+  });
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+dotenv.config();
+
+app.get("/ping", (req, res, next) => {
+  res.json({ message: "pong!" });
+});
+/////////////////////
+// # 2. 유저 회원가입 //
+/////////////////////
+app.post("/users", async (req, res) => {
+  const { id, name, email, profile_image, password } = req.body;
```

Comment:
```suggestion
  const { id, name, email, profileImageUrl, password } = req.body;
```
- JavaScript에서는 기본적으로 `snake_case`가 아닌 `CamelCase`로 작성합니다.
- 또한 snake_case, CamelCase를 뒤죽박죽 섞어서 사용하면 안됩니다.
- 하나의 컨벤션으로 통일해서 작성해 주세요.

Reply:
컨벤션 통일했습니다!